const { catchError } = require("./catchError");
const { MyError } = require("./myError");

module.exports = {
  catchError: catchError,
  throwError: (message, statusCode) => {
    throw MyError(message, statusCode);
  }
}
